### PR TITLE
protobuf support

### PR DESCRIPTION
**Pull request by s-Nick-s**
Regular endpoint is working, but the distance endpoint gives me
``SEC_E_UNTRUSTED_ROOT (0x80090325) - The certificate chain was issued by an authority that is not trusted. 
``
@EivydasKoc Is this endpoint working on your end?

### CODE REVIEW

**Comment by EivydasKoc** [comment]
Hi @s-Nick-s, distance endpoint is currently disabled so this is normal behaviour at the moment.

**Comment by EivydasKoc** [comment]
@s-Nick-s could you merge the master branch into this one and update the readme with a proto example as per other examples in the readme? We also have a complete readme description of proto  [here](https://github.com/traveltime-dev/traveltime-sdk-ruby#time-filter-fast-proto)

File: R/utils.R [hunk 1]
```diff
@@ -109,5 +136,8 @@ add_search_args <- function(...) {
   return(list(df))
 }
 
+encodeFixedPoints <- function(sourcePoint, targetPoints) {
```
Comment (EivydasKoc) [inline R/utils.R:139]: this should be a single `targetPoint`

File: R/zzz.R [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.onLoad <- function(libname, pkgname) {
```
Comment (EivydasKoc) [inline R/zzz.R:1]: Why is this file named so weird? Is this a leftover from development and debuging?

**Comment by EivydasKoc** [comment]
Overall this lgtm with my limited knowledge of R. I'll check this branch out and run some local tests.

File: R/utils.R (hunk 1 above)
Comment (s-Nick-s) [reply R/utils.R:139]: This encodes all target points at once, so no need to go over them one by one

File: R/zzz.R (hunk 2 above)
Comment (s-Nick-s) [reply R/zzz.R:1]: Standarrd naming convetion for [functions that load external files](https://r-pkgs.org/code.html#sec-code-r-landscape)

**Comment by EivydasKoc** [comment]
@s-Nick-s I'm having some trouble trying to run this locally , it says it cannot find the Transportation object even though the proto files are being read:

`Error during wrapup: error in evaluating the argument 'Class' in selecting a method for function 'new': object 'com.igeolise.traveltime.rabbitmq.requests.Transportation' not found
Error: no more error handlers available (recursive errors?); invoking 'abort' restart`

Could you give me some advice on how I should test this? Steps I have taken: 

1. open project in RStudio
2. make new r script
3. import `zzz.R ` `time-filter-fast-proto.R` and `data-raw/params.R` with `source()`
4. try run `time_filter_fast_proto` with [example](https://github.com/traveltime-dev/traveltime-sdk-r/pull/14/files#diff-60f2fe5c71bfe91660fa21c5ad8d3da8deb9fe9bdb408b298e90e81a10319be9R24) and get the error. I also tried to read proto files manually with the same result.

**Comment by EivydasKoc** [comment]
> @s-Nick-s I'm having some trouble trying to run this locally , it says it cannot find the Transportation object even though the proto files are being read:
> 
> `Error during wrapup: error in evaluating the argument 'Class' in selecting a method for function 'new': object 'com.igeolise.traveltime.rabbitmq.requests.Transportation' not found Error: no more error handlers available (recursive errors?); invoking 'abort' restart`
> 
> Could you give me some advice on how I should test this? Steps I have taken:
> 
> 1. open project in RStudio
> 2. make new r script
> 3. import `zzz.R ` `time-filter-fast-proto.R` and `data-raw/params.R` with `source()`
> 4. try run `time_filter_fast_proto` with [example](https://github.com/traveltime-dev/traveltime-sdk-r/pull/14/files#diff-60f2fe5c71bfe91660fa21c5ad8d3da8deb9fe9bdb408b298e90e81a10319be9R24) and get the error. I also tried to read proto files manually with the same result.

problem solved, reinstalled the package directly from this branch and it works as expected. Approved.

**Review by EivydasKoc — APPROVED** [review]

**Pull request merged**